bigtable-hbase-1.x-replication/pom.xml

```diff
@@ -181,6 +154,11 @@
             <!--       use the HBase server from the HBase environment-->
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud.bigtable</groupId>
+            <artifactId>bigtable-hbase-replication</artifactId>
+            <version>2.0.0-beta6</version>
+        </dependency>
         <dependency>
             <groupId>com.google.protobuf</groupId>
```

Comment:
Is it required to bring our own protobuf? If so, please shade it, we don't want our protobuf to conflict with hbase's version.